install/gcp/installer/forseti_installer.py

```diff
@@ -205,8 +205,7 @@ def preflight_checks(self):
             self.check_if_authed_user_in_domain(
                 self.organization_id, authed_user)
         else:
-            gcloud.activate_service_account(authed_user,
-                                            service_account_key_file)
+            gcloud.activate_service_account(service_account_key_file)
```

Comment:
I don't think so. The docs state that if you are providing a .json credential file, then there's no need to specify the authed user for the SA. This is because the credential file contained the authed user as key value pair.